build.gradle.kts

```
@@ -333,6 +333,7 @@ project(":") {
             exclude(module = "kotlin-stdlib")
             exclude(module = "kotlin-stdlib-common")
         }
+        implementation("com.vdurmont:semver4j:3.1.0")
```

```suggestion
        api("com.vdurmont:semver4j:3.1.0")
```
And then it's possible to remove this dependency from `toml` module